### ISSUE TITLE
Add an entry in the menu to generate LTC wave files

### ISSUE THREAD

**Issue by ycollet** [audacity/audacity#5418]
### Your idea

LTC signals are used to synchronize audio / video:
https://github.com/x42/ltc-tools

With a LTC wave file, you can send this signal to xjadeo and have an audio track from audacity synchronized with a video played by xjadeo. xjadeo is used by ardour to manage video.

Repository of xjadeo:
https://github.com/x42/xjadeo

### Problem to be solved

Synchronization of audio file with a video.

### Prior art

Ardour is using xjadeo and sync video via MTC (Midi Time Code) signals.

**Comment by jonata** [comment]
Can you explain more about this? Do you want a LTC Wave Generator inside Audacity, or you want to control/communicate with XJadeo somehow?

**Comment by ycollet** [comment]
For me, the kickest way to be able to control the position in a movie is to have a LTC track generated via audacity.
So, a LTC wave generator inside audacity is my first choice :)

**Comment by jonata** [comment]
Yes, that's what I want too. But I think the way we can do it is the same as Ardour, using MTC, as Audacity already have some portmidi architecture. Maybe I need to research more, but as fas as I understand, using a wave LTC would require sending the audio signal (this specific timecode wave) to somewhere. In the case of XJadeo, it should be send using Jack, but I'm not sure if that's available in all platforms. I will look into MTC.

On a related note, I made a pull request (https://github.com/audacity/audacity/pull/6152) to enable Audacity send the current position when requested. In this way, we could control an external player getting this information from Audacity. Not the best solution, though.

**Comment by jonata** [comment]
Good news! Yes, MTC works:

[Gravação de tela de 2024-03-22 00-00-17.webm](https://github.com/audacity/audacity/assets/908065/0ea17c1f-8780-464f-9d22-524613eb306b)

Now I'll figure out how to integrate it in Audacity properly.

**Comment by ycollet** [comment]
Excellent ! Thanks a lot !

**Comment by devmaiter** [comment]
Hi, I'd like to work on this if it's still open
